scripts/apps/authoring/metadata/views/metadata-dropdown.html

```diff
@@ -1,16 +1,16 @@
 <div class="dropdown dropdown--noarrow dropdown--dropleft" dropdown sd-metadropdown-focus>
-    <!-- if field is other than 'place', 'genre', 'anpa_category', 'subject', 'authors'-->
-    <button ng-disabled="disabled" class="dropdown__toggle line-input" tabindex="{{tabindex}}" dropdown__toggle ng-if="listFields.indexOf(field) === -1 && !icon">
+    <!-- if field is single input field-->
+    <button ng-disabled="disabled" class="dropdown__toggle line-input" tabindex="{{tabindex}}" dropdown__toggle ng-if="multiInputFields.indexOf(field) === -1 && !icon">
         {{ key && values[item[field]]  && values[item[field]].name ? values[item[field]].name: (item[field].name || item[field] ? item[field].name || item[field] : "none") | translate }}
         <b class="dropdown__caret"></b>
     </button>
-    <!-- if field is one of 'place', 'genre', 'anpa_category', 'subject', 'authors' and no cv-->
-    <button ng-disabled="disabled" class="dropdown__toggle line-input" tabindex="{{tabindex}}" dropdown__toggle ng-if="listFields.indexOf(field) > -1 && !icon && !cv">
+    <!-- if field is multi input field and no cv-->
```

Comment:
> and no cv
is not that clear. I'd prefer "and the data source is not a custom vocabulary"

What other data sources are supported, free text input?

Reply:
It supports custom vocabulary only. But there are some CVS which are stored inside multiInputFields and some create their own field in the item.